logstash-core/lib/logstash/api/lib/app.rb

```diff
@@ -25,6 +25,19 @@ def initialize(app=nil)
       @factory = CommandFactory.new(settings.service)
     end
 
+    get "/_directory" do
+      data = {}
+      ROUTES.each_pair do |namespace, _routes|
+        routes = {}
+        _routes.each_pair do |verb, methods|
+          next if "HEAD" == verb
+          routes[verb] = methods.map { |m| "http://localhost:9600#{namespace}/#{m[0].source.gsub("\\A/","").gsub("\\z", "")}" }
```

Comment:
This doesn't take into consideration the user configuration of the host and port?